src/components/va-accordion/va-accordion.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { html } from 'lit-html';
+
+export default {
+  title: 'Components/Accordion',
+  component: 'va-accordion',
+};
+
+const defaultArgs = {
+  multi: false,
+  bordered: false,
+};
+
+const argTypes = {
+  first: {
+    description: 'The first name',
+  },
+  middle: {
+    description: 'The middle name',
+  },
+  last: {
+    description: 'The last name',
+  },
+};
+
+const Template = ({ multi, bordered }) => html`
```

Comment:
These stories right now are only used for local development. We may explore exporting these and importing them into the storybook in `master` so that we don't have to have duplicate code.